.github/workflows/ci_tests_linux.yml

```diff
@@ -16,3 +17,5 @@ jobs:
       run: .github/scripts/install_software.sh
     - name: Check-in Tests
       run: sudo bash -c ".github/scripts/ci_tests.sh"
+    - name: Build Esp32 Sample
+      run: docker run -v "${PWD}:/iot-middleware-freertos-samples" -w /iot-middleware-freertos-samples espressif/idf .github/scripts/ci_tests.sh -esp
```

Comment:
>/iot-middleware-freertos-samples espressif/idf [](http://example.com/codeflow?start=70&length=46)

working directory should be same to your mounting directory? or I am missing something here?

docker run -v "${PWD}:/iot-middleware-freertos-samples" -w /iot-middleware-freertos-samples .github/scripts/ci_tests.sh -esp

Reply:
I'm not sure what you are proposing. 
Your command is missing the image name. The one that is in the code is correct.